callstats-java-sdk/src/main/java/io/callstats/sdk/messages/BridgeKeepAliveMessage.java

```diff
@@ -10,144 +10,78 @@
  * The Class BridgeKeepAliveMessage.
  */
 public class BridgeKeepAliveMessage {
-	
-	/** The app id. */
-	private int appID;
-	
 	/** The bridge id. */
-	private String bridgeID;
+	private String localID;
 	
-	/** The version. */
-	private String version;
+	/** The timestamp. */
+	private long timestamp;
 	
-	/** The api ts. */
-	private long apiTS;
+	/** The origin id. */
+	private String originID;
 	
-	/** The token. */
-	private String token;
+	/** The device id. */
+	private String deviceID;
 	
 	/** The logger. */
 	private static final Logger logger = LogManager.getLogger("CallStats");
 		
 	/**
 	 * Instantiates a new bridge keep alive message.
 	 *
-	 * @param appID the app id
 	 * @param bridgeID the bridge id
-	 * @param version the version
-	 * @param apiTS the api ts
-	 * @param token the token 
+	 * @param timestamp the api ts
+	 * 
 	 */
-	public BridgeKeepAliveMessage(int appID, String bridgeID,
-			String version, long apiTS, String token) {
-		super();
-		this.appID = appID;
-		this.version = version;
```

Comment:
We should make sure this is sent somewhere, otherwise we won't know which versions of the library are in use

Reply:
yes it will be going in endpoint info.